src/styles/_catalog.scss

```diff
@@ -1,22 +1,21 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $card-width);
+  justify-content: center;
+  gap: $card-gap;
 
   max-width: 944px;
```

Comment:
Once Ihor told us that it's better to start with mobile media-query

Reply:
Oh, I decided to use desktop-first in this task:)